jwst/outlier_detection/utils.py

```diff
@@ -30,20 +31,112 @@ def create_cube_median(cube_model, maskpt):
     return median
 
 
-def create_median(resampled_models, maskpt):
+def create_median(resampled_models, maskpt, on_disk=True, buffer_size=10.0):
     """Create a median image from the singly resampled images.
+    resampled_models is expected to be a ModelLibrary for imaging modes.
```

Comment:
Would you add `buffer_size` (and perhaps the other arguments) to this docstring? I think the non-`buffer_size` ones are pretty obvious so maybe not needed for this private function.

Reply:
I put something minimal in.  Note that this will likely change with [JP-3706](https://jira.stsci.edu/browse/JP-3706)